tests/integration_tests/functional/test_cpu_features_aarch64.py

```diff
@@ -87,39 +108,47 @@ def test_default_cpu_features(test_microvm_with_api):
     vm.basic_config()
     vm.add_net_iface()
     vm.start()
-    _check_cpu_features_arm(vm)
+    _check_cpu_features_arm(vm, "4.14")
```

Comment:
This fixture should be using a 5.10 kernel

Reply:
Updated this test to use `microvm_factory` like other tests